cves/kernel/CVE-2021-3659.yml

```diff
@@ -399,8 +404,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: "need to check that the input wasn't null instead of trusting it"
```

Comment:
I like the connection to distrust input, but it is not user input here. Try to elaborate about what the input is and that it's a parameter that was input by another function. Shows that checks on code created inputs should also be checked and not just external user inputs.